src/content/1.1/code/agda/snippet04.agda

```diff
@@ -1,4 +1,10 @@
-f : A -> B
-g : B -> C
-h : C -> D
-h ∘ (g ∘ f) == (h ∘ g) ∘ f == h ∘ g ∘ f
\ No newline at end of file
+open import Function using (_∘_)
+open import Relation.Binary.PropositionalEquality using (_≡_)
+
+_ :  {A B C D : Set}
+     (f : A → B)
+     (g : B → C)
+     (h : C → D)
+  →  (h ∘ g) ∘ f ≡ h ∘ (g ∘ f)
+
+_ = λ f g h → _≡_.refl
```

Comment:
I wonder if explicitly importing refl would be a bit easier for readers:
```agda
open import Function using (_∘_)
open import Relation.Binary.PropositionalEquality using (_≡_; refl)

_ :  {A B C D : Set}
     (f : A → B)
     (g : B → C)
     (h : C → D)
  →  (h ∘ g) ∘ f ≡ h ∘ (g ∘ f)

_ = λ f g h → refl
```

Reply:
Yes, let's change to that convention (i.e., we should import refl).  By the way, it appears the composition operator is working for you now... yes?